dls-controls/malcolmjs
# Feature/malcolm reducer refactor #207

## Description

Refactored malcolm reducer

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm test
- check everything passes

## Agile board tracking

connect to #207 
closes #207
